src/core/00_github_runner.tf

```
@@ -30,3 +30,96 @@ module "github_runner" {
 
   tags = var.tags
 }
+
+locals {
+  repo_owner = "PagoPA"
+  repo_name  = "io-infra"
+  image_name = "ghcr.io/pagopa/github-self-hosted-runner-azure:beta-runner-v2"
+}
+
+data "azurerm_key_vault" "keyvault" {
+  name                = "io-p-kv-common"
+  resource_group_name = "io-p-rg-common"
+}
+
+data "azurerm_key_vault_secret" "github_pat" {
+  name         = "github-pat-io-infra"
+  key_vault_id = data.azurerm_key_vault.keyvault.id
+}
+
+resource "azapi_resource" "job" {
+  type      = "Microsoft.App/jobs@2023-05-01"
+  name      = "io-infra-github-runner-job"
```

```suggestion
  name      = "${local.project}-infra-github-runner-job"
```
use local project prefix